WikiWatershed/model-my-watershed
# Add protected lands overlay to layer selector

## Overview

This PR adds the protected-lands-30m overlay option to the layer selector along with a legend.

Connects #2951 
Connects #2952 
Connects #2953 

### Demo

![screen shot 2018-09-18 at 11 10 04 am](https://user-images.githubusercontent.com/4165523/45697447-d8cf7180-bb33-11e8-96e7-d10ca750a508.png)

## Testing Instructions

- get this branch, `bundle.sh --vendor`, then visit localhost:8000
- select the "Protected Lands" overlay from the layer selector and verify that
    - the overlay is visible
    - the opacity slider works
    - the legend matches the color map here https://github.com/WikiWatershed/model-my-watershed/issues/2951#issuecomment-421442719
